### PR TITLE
Сhanged elastic.io mentions to neutral ones

### PR DESCRIPTION
**Pull request by pnedelko**
Removed elastic.io mentions to be more whitelabel friendly.

### CODE REVIEW

File: source/includes/_components.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ There are three sharing modes:
 
 - ``team`` – no sharing. Only Contract members can use the component.
 - ``tenant`` – component could be used by the other clients in the tenant.
-- ``global`` – special mode for components from the standard set of components of Elastic.io Platform (e.g. ``Timer``, ``Webhook`` etc). Any user of the platform can use ``global`` components.
+- ``global`` – special mode for components from the standard set of components of the Platform (e.g. ``Timer``, ``Webhook`` etc). Any user of the platform can use ``global`` components.
```
Comment (honeybadgeralex) [inline source/includes/_components.md]: a special mode for Сomponents from the standard Component set of the Platform (e.g. ``Timer``, ``Webhook`` etc). Any Platform user can use ``global`` components.

File: source/includes/_components.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ There are three sharing modes:
 
 - ``team`` – no sharing. Only Contract members can use the component.
 - ``tenant`` – component could be used by the other clients in the tenant.
```
Comment (honeybadgeralex) [inline source/includes/_components.md]: the Component can be used by the other clients in the same Tenant.

File: source/includes/_contracts.md [hunk 3]
```diff
@@ -281,7 +281,7 @@ Content-Type: application/json
         "roles":[
           "admin"
         ],
-        "email":"alla.ospik@{{ product_name }}"
+        "email":"alla.ospik@example.com"
```
Comment (honeybadgeralex) [inline source/includes/_contracts.md]: jane.doe@example.com

File: source/includes/_tenants.md [hunk 4]
```diff
@@ -544,7 +544,7 @@ This request is authorized for the users with the `tenants.tenant.edit` permissi
 | attributes.custom_scripts\[]             | no       | Customer js-scripts.      |
 | attributes.default_workspace_type       | no       | The type of Workspaces which will be created in given Tenant. The value must be `full` or `limited`        |
 
-_Note_: If Tenant's domains are matches to the `*.elastic.io` (where `*` can not contain `.`) then given Tenants can use the default Certificates. To remove existed Certificates, specify them as null (e.g. `"app": null`)
+_Note_: If the default installation's certificate is a wildcard one (e.g. `*.example.com`) and the Tenant's domains match to this certificate (e.g. `my-tenant-api.example.com`), then there is no need to provide separate certificates for these domains. To remove existing certificates set them to `null` (e.g. `"app": null`).
```
Comment (honeybadgeralex) [inline source/includes/_tenants.md]: remove existing certificates, set them to `null` (e.g. `"app": null`).

File: source/includes/_components.md (hunk 1 above)
Comment (pnedelko) [reply source/includes/_components.md]: done

File: source/includes/_components.md (hunk 2 above)
Comment (pnedelko) [reply source/includes/_components.md]: done

File: source/includes/_contracts.md (hunk 3 above)
Comment (pnedelko) [reply source/includes/_contracts.md]: done

File: source/includes/_tenants.md (hunk 4 above)
Comment (pnedelko) [reply source/includes/_tenants.md]: done

**Pull request merged**